tests/integration/testdata/buildcmd/terraform/zip_based_lambda_functions_s3_backend/artifacts/layer1/requirements.txt

```diff
@@ -0,0 +1,2 @@
+boto3
+ptvsd
```

Comment:
why ptvsd? its for debugging right?

Reply:
I just add some dependencies, and then imported in the python code to make sure that the build is really working.

Reply:
and actually it seems no used in the library code, I will remove it.